reference/tools/gnu/autotoolstoolchain.rst

```diff
@@ -236,6 +236,8 @@ For instance:
             })
             at.generate()
 
+The ``AutotoolsToolchain`` will listen to ``tools.gnu:extra_configure_args`` from the :ref:`reference_config_files_global_conf` to extend the
+``configure_args`` attribute.
```

Comment:
I think it's fine to add a comment here, saying that you can add your own configure args with this conf but this should also go down below in the conf section

Reply:
Done, please, see the commit [2fe9006](https://github.com/conan-io/docs/pull/4100/commits/2fe9006372a929beed9c479e5e8d3649a5134b66)